inlong-dashboard/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@ant-design/icons": "^5.0.1",
     "@ant-design/pro-layout": "^7.10.3",
+    "@babel/traverse": "^7.23.2",
```

Comment:
Do not add it to dependencies because the project does not directly depend on it.